src/Compute/Compute.Autorest/docs/Update-AzGalleryApplicationVersion.md

```diff
@@ -15,8 +15,18 @@ Update a gallery Application Version.
 ### UpdateExpanded (Default)
 ```
 Update-AzGalleryApplicationVersion -GalleryApplicationName <String> -GalleryName <String> -Name <String>
- -ResourceGroupName <String> -PackageFileLink <String> [-SubscriptionId <String>]
- [-DefaultConfigFileLink <String>] [-PublishingProfileEndOfLifeDate <DateTime>]
+ -ResourceGroupName <String> -PackageFileLink <String> [-SubscriptionId <String>] [-ConfigFileName <String>]
```

Comment:
Two new parameters added - ConfigFileName, PackageFileName?

Reply:
I removed this two parameters, because these two parameters were also generated in the previous version, but they were not displayed to users.